### DISCUSSION TITLE
PS4 to Genesis Acting like 3 button controller

### DISCUSSION THREAD

**Discussion by lamprey13** [darthcloud/BlueRetro#998] [Q&A]
Hi, my receiver is running the latest firmware 24.04. I have the receiver setup as GamepadAlt mode. I can conncet no problem and the contoller is registering inputs, but on the Genesis no X, Y or Z buttons get triggered no matter how I map things. 

It appears to be acting like a 3 button controller. Is there a setting that I need to set for the PS4 controller to act like a 6 button controller?

**Comment by lamprey13** [comment]
Just an FYI, I reverted to v1.9 and everythig works, so it looks like a bug was introduced in v24.04

**Comment by darthcloud** [comment]
Try again on v25.01, if still not working you should open an issue
